integration-cli/docker_api_exec_test.go

```diff
@@ -108,6 +99,30 @@ func (s *DockerSuite) TestExecAPIStartBackwardsCompatible(c *check.C) {
 	c.Assert(resp.StatusCode, checker.Equals, http.StatusOK, comment)
 }
 
+// #19362
```

Comment:
See it as a nit, but in general I think code should not have to rely on GitHub in order to get the information needed.